server/maintenance_requests.go

```diff
@@ -53,4 +54,14 @@ func (app *App) CreateMaintenanceRequest(c *gin.Context) {
 		return
 	}
 	c.JSON(http.StatusOK, gin.H{"id": req.ID})
-}
\ No newline at end of file
+}
+
```

Comment:
Can you add a comment with a brief description of the function?

Reference: https://blog.golang.org/godoc